scripts/services/getConfig.ts

```diff
@@ -21,24 +21,15 @@ export const getConfig: T.ExpectedExports.getConfig = compat.getConfig({
     "target": "lan-address",
     "interface": "main",
   },
-  "wallet": {
-    "type": "union",
+  "implementation": {
+    "type": "enum",
     "name": "Lightning Implementation",
-    "description": "LNBits Lightning Implementation",
-    "tag": {
-        "id": "type",
-        "name": "Select Lightning Node",
-        "variant-names": {
-            "LndRestWallet": "Lightning Network Daemon",
-            "CLightningWallet": "Core Lightning",
-        },
-        "description":
-            "The LN node to connect to",
-        },
-    "default": "LndRestWallet",
-    "variants": {
-      "LndRestWallet": {},
-      "CLightningWallet":{},
-    }
+    "description": "The underlying Lightning implementation, currently LND or Core Lightning (CLN)",
+    "values": ["lnd", "cln"],
```

Comment:
```
# Choose from LNPayWallet, OpenNodeWallet, LntxbotWallet, ClicheWallet, LnTipsWallet
#             LndRestWallet, CoreLightningWallet, LNbitsWallet, SparkWallet, FakeWallet, EclairWallet
LNBITS_BACKEND_WALLET_CLASS=LndRestWallet
```
changing the values to `lnd` and `cln` will break the `.env` file

Reply:
Ok, that seems like an SDK bug, but I pushed a fix that shouls satisfy it